pkg/contracts/script/BaseSetup.sol

```diff
@@ -46,6 +46,7 @@ contract BaseSetup is SetupScript {
     function setUp() public virtual {
         // create forks
         goerliFork = vm.createFork(GOERLI_RPC_URL);
+        vm.selectFork(goerliFork);
```

Comment:
Is important to choose the fork right away so our contracts exists on the forked chain.